docs/static/resources/openapi.json

```diff
@@ -15618,7 +15618,7 @@
                       "description": "Menu items in a forest like data structure",
                       "items": {
                         "properties": {
-                          "childs": {
+                          "children": {
```

Comment:
Isn't this file programmatically generated?

Reply:
It appears to be updated by commits that change things: https://github.com/apache/superset/commit/02ef9ca4cd60d0d05f0afd2cebc1cabb83dfd4df

The relevant items should be changed below...

Reply:
unfortunately I think this is part of the api, you would have to update flask appbuilder to fix it and it would be a breaking change. @dpgaspar

Reply:
I'm quite happy to drop anything that proves to be problematic. It's not a big deal.

And in the interest of moving things forward while I'm away, I'll drop this now.

You're welcome to fix it w/o me, or I can look at it once the rest merges.

Reply:
https://github.com/apache/superset/blob/3313530f4de2d1cf1719d6b2beda89a5bfbba2cd/superset/cli/update.py#L148-L152

This file is generated by a script. There is a superset command: `superset update-api-docs` but unfortunately it is not documented anywhere.